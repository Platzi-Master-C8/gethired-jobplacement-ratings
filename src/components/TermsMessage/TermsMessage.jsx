import React from 'react';
import { Typography } from '@mui/material';

import { TermsConditiosModal } from '../TermsConditiosModal';
import { PrivacyPolicyModal } from '../PrivacyPolicyModal';

const TermsMessage = () => (
    <Typography variant="caption" sx={{ display: 'flex', flexWrap: 'wrap', wordWrap: 'break-word' }}>
        Al enviar este formulario estas aceptando nuestros <TermsConditiosModal /> y la <PrivacyPolicyModal />
    </Typography>
);

export default TermsMessage;
