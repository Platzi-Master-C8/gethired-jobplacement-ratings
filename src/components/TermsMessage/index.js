import TermsMessage from './TermsMessage';

export { TermsMessage };
