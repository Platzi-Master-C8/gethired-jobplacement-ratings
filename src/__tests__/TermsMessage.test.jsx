import React from 'react';

import { render } from '@testing-library/react';
import { TermsMessage } from 'Components/TermsMessage';

describe('Component <TermsMessage/>', () => {
    test('renders TermsMessage', () => {
        const { asFragment } = render(<TermsMessage />);
        expect(asFragment()).toMatchSnapshot();
    });
});
