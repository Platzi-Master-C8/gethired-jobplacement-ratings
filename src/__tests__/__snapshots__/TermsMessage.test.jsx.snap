// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <TermsMessage/> renders TermsMessage 1`] = `
<DocumentFragment>
  <span
    class="MuiTypography-root MuiTypography-caption css-1i6gjti-MuiTypography-root"
  >
    Al enviar este formulario estas aceptando nuestros 
    <div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12 css-oav0mg-MuiGrid-root"
      >
        <a
          class="MuiTypography-root MuiTypography-caption MuiLink-root MuiLink-underlineNone css-v181wt-MuiTypography-root-MuiLink-root"
        >
          Terminos y Condiciones
        </a>
      </div>
    </div>
     y la 
    <div>
      <div
        class="MuiGrid-root MuiGrid-item MuiGrid-grid-md-12 css-oav0mg-MuiGrid-root"
      >
        <a
          class="MuiTypography-root MuiTypography-caption MuiLink-root MuiLink-underlineNone css-v181wt-MuiTypography-root-MuiLink-root"
        >
          Politica de Privacidad
        </a>
      </div>
    </div>
  </span>
</DocumentFragment>
`;
